# State general
robotnik_msgs/State state
# Substate
string substate
# Current group of joints being controlled
string current_group
# Selected TCP
string tcp
# Control mode (POSITION, VELOCITY)
string control_mode
# CONTROL TYPE (CARTESIAN_EULER, JOINT_BY_JOINT, TRAJECTORY)
string control_type
# Current Goal state 
string goal_state
